layouts/shortcodes/charter.html

```diff
@@ -1,8 +1,13 @@
-{{ $headers := dict }}
+{{ $url := "https://api.github.com/repos/maplibre/maplibre/contents/CHARTER.md?ref=main" }}
+{{ $cacheKey := print $url (now.Format "2006-01-02") }}
```

Comment:
why I added this: https://gohugo.io/functions/resources/getremote/#caching

Given that the site is build in CI might not be a real issue, but (if the charter gets changed in the future) might prevent dev-confusion why this is not updated..